app/views/questions/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Recent Questions</h1>
+
+<table class="table table-striped">
+	<thead>
+		<tr>
+			<th>Question</th>
+			<th>Inquirer</th>
+			<th>Answers</th>
+		</tr>
+	</thead>
+	<tbody>
+		<%@questions.each do |question|%>
+		<tr>
+			<td><h4><%= link_to "#{question.title}", question_path(question) %></h4></td>
+			<td><h4><%= link_to "#{question.user.name}", user_path(question.user_id) %></h4></td>
```

Comment:
Be aware that the line:

```ruby
question.user.name
```
will cause a db hit unless you used:

```ruby
Question.find(...).includes(:user)
```
when populating your question object.